cargo-miri/src/phases.rs

```diff
@@ -528,7 +528,7 @@ pub fn phase_runner(mut binary_args: impl Iterator<Item = String>, phase: Runner
     cmd.args(binary_args);
 
     // Make sure we use the build-time working directory for interpreting Miri/rustc arguments.
-    // But then we need to switch to the run-time one, which we instruct Miri do do by setting `MIRI_CWD`.
+    // But then we need to switch to the run-time one, which we instruct Miri do by setting `MIRI_CWD`.
```

Comment:
This should have been "to do". the PR replaced one type by another. :)

Could you file a follow-up to fix that?

Reply:
Haha, sure.